audtorch/transforms/functional.py

```diff
@@ -256,10 +256,50 @@ def normalize(signal, *, axis=None):
     return signal / np.maximum(peak, 1e-7)
 
 
-def stft(signal, window_size, hop_size, *, window='hann', axis=-1):
-    r"""Short-term Fourier transform.
+def standardize(signal, axis=None, mean=True, std=True):
```

Comment:
I can see the need for flexibility. Being a bit picky but should not the naming `standardize` by default entail `mean=True` and `std=True`?

Reply:
I think, the compactness of this implementation outweighs the fact that it's slightly counter-intuitive.

Reply:
It is the standard behavior people are used to by `sklearn`'s `StandardScaler`. Plus standardization usually means `mean & std` normalization. Just doing mean normalization would probably be referred to as `mean centering` for example. So I see no need to change the name or demand that the user always specifies the mean and std arguments.

Reply:
@phtephanx if you are ok with the current implementation can you mark this as resolved please.